generator/src/main/java/com/scottlogic/deg/generator/visualisation/VisualisationConfigSource.java

```diff
@@ -0,0 +1,13 @@
+package com.scottlogic.deg.generator.visualisation;
+
+import com.scottlogic.deg.generator.ConfigSource;
+
+/**
+ * Command line options specific to visualisation of a profile.
+ */
+public interface VisualisationConfigSource extends ConfigSource {
+
+    String getTitleOverride();
```

Comment:
I'm not sure what Title means here, perhaps documenting the interface would help?

Reply:
to document it, I would need to know what it is too

Reply:
Title override allows a user to specify a title to appear in the visualised graph, rather than taking the description of the profile (or the file name if no description is present).

Provided by the `--title` switch - see [Visualise options](https://github.com/ScottLogic/datahelix/blob/master/docs/Options/VisualiseOptions.md)

Reply:
added javadoc link to documentation file.